tools/migrations/24-05-update_learning_cycle_for_existing_users.sql

```diff
@@ -0,0 +1,6 @@
+/* 
```

Comment:
let's try to use the following naming convention from today on: https://github.com/zeeguu/api/blob/master/tools/migrations/README.md#convention-for-naming-migrations